atlp-rwanda/wolves-bn-backend
# #174772155 Setup the basic folder structure for the entire project

#### What does this PR do?
Setup the basic folder structure of the project
#### Description of Task to be completed?
Have a basic folder structure for the team to get everyone started. It should have models, database, required npm packages, routes.
#### How should this be manually tested?
After cloning the repo, run npm install to install dependencies and npm start to start the server.
Check the folder structure and if es6 works.
#### What are the relevant pivotal tracker stories?
#174772155
